lib/Dialect/ESI/ESIOps.cpp

```diff
@@ -12,8 +12,11 @@
 using namespace mlir;
 using namespace circt::esi;
 
-//====== Channel buffer.
+namespace {
 
+// ====
+// ChannelBuffer methods.
+// ====
 ParseResult parseChannelBuffer(OpAsmParser &parser, OperationState &result) {
```

Comment:
Please mark these static instead of using a large anonymous namespace:
https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

Reply:
Also I'd recommend using the standard file divider comment.